configs/stylelint-config-spirit/README.md

```diff
@@ -0,0 +1,37 @@
+# stylelint-config-spirit
+
+> StyleLint configuration and preset for Spirit Design System
+
+## Getting started
+
+This preset is part of this monorepo only and thus it is accessible for every package.
+
+To install `stylelint-config-spirit` in your project, you will need to run the following command using [Yarn][yarn]:
+
+```bash
+yarn add stylelint-config-spirit
+```
+
+## 🚀 Usage
+
+```js
+// stylelint.config.js
```

Comment:
maybe
```suggestion
// .stylelintrc.mjs
```
like we actually use it?

Reply:
This file can have multiple names, I will reword it to something like `Stylelint Configuration File`